client/main/main.js

```diff
@@ -317,7 +331,6 @@ $(document).ready(() => {
             return instance.open(toOpen);
 
     }).catch(() => { // if the initial open fails
-
```

Comment:
a bit passive aggressive? :P

Reply:
ah! this was a by product of me putting console.logs in here while debugging and then i removed the lines (as well as what was previously a blank line).